src/python/grpcio/grpc/aio/_base_server.py

```diff
@@ -295,3 +295,13 @@ def auth_context(self) -> Mapping[str, Iterable[bytes]]:
         Returns:
           A map of strings to an iterable of bytes for each auth property.
         """
+
+    @abc.abstractmethod
```

Comment:
Is it backwards compatible to add this with an `abstractmethod` annotation now?

Reply:
Removed. Even though this is provided in sync, maybe we should not break backward compatibility.